zebra-consensus/src/block/subsidies.rs

```diff
@@ -0,0 +1,241 @@
+//! Calculation of Block Subsidy, Funding Streams, and Founders’ Reward
+
+use super::*;
+
+use zebra_chain::{
+    block::Height,
+    parameters::{Network, NetworkUpgrade::*},
+};
+
+use crate::parameters::{fs, Params};
+
+fn slow_start_shift() -> Height {
+    Height(Params::SLOW_START_INTERVAL / 2)
+}
+
+fn slow_start_rate() -> u32 {
+    Params::MAX_BLOCK_SUBSIDY / Params::SLOW_START_INTERVAL
+}
+
+fn is_bolossom_activated(height: block::Height, network: Network) -> Option<Height> {
+    let blossom_height = Blossom.activation_height(network)?;
+    let condition = height >= blossom_height;
+    match condition {
+        true => Some(blossom_height),
+        false => None,
+    }
+}
```

Comment:
I think this function isn't really clear what it does and would be better if just inlined and simplified using the `filter` fn on `option`

```rust
let blossom_height = Blossom
    .activation_height(network)
    .filter(|blossom_height| height >= blossom_height);
```

Reply:
Updated to this at https://github.com/ZcashFoundation/zebra/pull/1051/commits/f3f6ec63110fac434263c0c9c728a54b27b8deb0